recipe/meta.yaml

```diff
@@ -14,24 +14,27 @@ source:
     sha256: {{ sha256 }}
 
 build:
-    number: 1
-    skip: true  # [win and not py36]
+    number: 1001
 
 requirements:
     build:
         - {{ compiler('cxx') }}
+        - {{ cdt('mesa-libgl-devel') }}  # [linux]
```

Comment:
Should this be in host?

Reply:
Thanks for reviewing. Hmm it's in the build section for the anaconda libglu too. So I guess it's correct. But maybe @mingwandroid can give some insights on which environment should be selected for cdt's.

Reply:
I am not super familiar with CDTs, but I would guess that build is probably right then

Reply:
ok, thanks,  so I think I am going to merge this.